#68 allow namespace CSS class

efx opened the pull request:
This will allow users to disable any of the supplied animations by instantiating the component like so: 
```
{{ember-notify classPrefix=“custom-namespace”}}
```
I will polish this documentation for inclusion in the README once the PR is finalized.

aexmachina commented:
All good, just needs docs. Nice one.

efx commented:
Thank you! Added some basic documentation for how to acheive custom positioning/animation. I'm working on custom styles for a project, so I can possibly provide those as an example file for a more detailed styling example.

aexmachina commented:
Yes that'd be good, thanks.
On Tue, 24 Nov 2015 at 01:25, Eli Flanagan <notifications@github.com> wrote:

> Thank you! Added some basic documentation for how to acheive custom
> positioning/animation. I'm working on custom styles for a project, so I can
> possibly provide those as an example file for a more detailed styling
> example.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aexmachina/ember-notify/pull/70#issuecomment-158948576>
> .
>

efx commented:
I added an example file for custom positioning and animation and updated the docs. Depending on demand or anticipated usage, I can publish the LESS file for that styling as an addon or something.

aexmachina commented:
Nice one, thanks.

aexmachina commented:
Published v5.2.0

efx commented:
my pleasure. it is a helpful plugin, so thanks for publishing it.
